cmake/faceliftMacros.cmake

```diff
@@ -405,8 +407,13 @@ macro(_facelift_add_target_start IMPLEMENTATION_TARGET_NAME)
     set(HEADERS_TO_BE_MOCCED ${HEADERS} ${HEADERS_NO_INSTALL})
     unset(HEADERS_MOCS)
 
+    # exclude unnecessary files mocking
+    foreach(HEADER ${HEADERS_NO_MOC})
+        string(REPLACE "${HEADER}" "" HEADERS_TO_BE_MOCCED "${HEADERS_TO_BE_MOCCED}")
```

Comment:
HEADERS_TO_BE_MOCCED sounds weird :) If you could find better name it would be beneficial (otherwise keep it as is)

Reply:
it's the existing variable, I try to change as little as possible, because this part is very dangerous.
Will be changed though.